packages/server/src/server.ts

```diff
@@ -39,12 +38,11 @@ const mainLogger = pino({
   }
 });
 
+const cors = require('koa-cors');
 const logger = mainLogger.child({ name: 'rpc-server' });
 const register = new Registry();
 const relay: Relay = new RelayImpl(logger, register);
-const cors = require('koa-cors');
-const app = new Koa();
-const rpc = new KoaJsonRpc();
+const app = new KoaJsonRpc();
```

Comment:
Maybe in a different variable (first one to be `const rpc = new KoaJsonRpc()` , and `const app = rpc.getKoaApp()` ), because this one would be pointing to the koa directly and we would be able to access other methods in koaJsonRpc - (`useRpc` and `rpcApp`). 
Server.ts and koaJsonRpc/index.ts can be in one file. My only concern for not joining them was, if it wouldn't be too hard to maintain because of a lot of code in one place. But if you think that it won't be a problem, I can join them ?

Reply:
Gotcha.
I'm okay with the separation then.
Thanks